builtin/game/static_spawn.lua

```diff
@@ -1,17 +1,13 @@
 -- Minetest: builtin/static_spawn.lua
 
-local function warn_invalid_static_spawnpoint()
-	if core.setting_get("static_spawnpoint") and
-			not core.setting_get_pos("static_spawnpoint") then
-		core.log("error", "The static_spawnpoint setting is invalid: \""..
-				core.setting_get("static_spawnpoint").."\"")
-	end
+if core.setting_get"static_spawnpoint" and
+		not core.setting_get_pos"static_spawnpoint" then
```

Comment:
Even if this is valid Lua, it doesn't fit into this file. The coding style should be the same everywhere.